tests/test_activity.py

```diff
@@ -561,9 +561,11 @@ def task_a(value):
         execution_context=dict(somemore='values'))
     resp = instance.create_execution_input()
 
-    assert len(resp) == 2
+    assert len(resp) == 4
     assert resp.get('context') == 'yes'
-    assert resp.get('somemore') == 'values'
+    assert 'execution.domain' in resp
+    assert 'execution.run_id' in resp
+    assert 'execution.workflow_id' in resp
 
```

Comment:
oof, my `assert resp.get('somemore') == 'values'` tests for the opposite of this.

maybe add instead
```
assert 'somemore' not in resp
assert 'unused' not in resp
```